.travis.yml

```diff
@@ -63,7 +63,7 @@ jobs:
       env: CODING_STANDARDS
       php: 7.1
       install:
-        - travis_retry composer require -n --prefer-dist --dev doctrine/coding-standard:^4.0
+        - travis_retry composer require -n --prefer-dist --dev doctrine/coding-standard:^5.0
```

Comment:
Is there some reason why we don't require it in the `composer.json` itself?

Reply:
I think it is probably just old and can be updated to match other repos.

Reply:
The reason is that DoctrineBundle sadly still supports dead PHP versions.